docs/developer/decision-records/2022-10-20-gaiax-bundle-repository/README.md

```diff
@@ -0,0 +1,28 @@
+# Gaia-X Bundle Repository
+
+## Decision
+
+A dedicated Gaia-X Bundle repository will be created in the [EDC project](https://github.com/eclipse-dataspaceconnector).
+This repository will contain extensions and relevant documentation enabling the deployment and operation of EDC components in compliance with the
+[Gaia-X Trust Framework](https://gaia-x.eu/wp-content/uploads/2022/05/Gaia-X-Trust-Framework-22.04.pdf).
+
+## Rationale
+
+Enabling sovereign and secured data exchanges between companies implies to be compliant with a set of rules.
+
+The Gaia-X Trust Framework is today the industry standard in terms of sovereign and trusted data exchanges which defines
+a set of rules known as the Compliance Process. Every Gaia-X certified dataspace must be compliant with these rules in order
+to enable trust and interoperability. This implies some constraints on how the participant's data exchange components are deployed and operated in such Gaia-X dataspace, e.g. identity enforcement, access control...
+
+In order to ease adoption of the EDC into Gaia-X dataspaces, it was decided to create a dedicated _Gaia-X Bundle_ repository under the EDC project, which will
+contain all extensions and relevant documentation for deploying and using the EDC with respect of the Gaia-X Trust Framework.
+These extensions will be packaged and published in the same way as the other EDC components, making it easy to build a Gaia-X compliant version of the EDC.
+
+> ⚠️ The EDC project does not aim at supporting all trust frameworks that might be developed at a later stage, and that might be or not based on Gaia-X.
+> Thus, only the present Gaia-X Bundle repository will be created and maintained under the EDC project. Any other future framework would have to
+> develop and maintain its own set of dedicated extensions.
+
```

Comment:
The EDC project does not aim to support all future trust frameworks in a similar form and make them available as bundles as comparable artifacts. Compliance with Gaia-X was stated as a goal in the proposal on the part of the EDC project and takes a separate role in the consideration. 
Thus, only the present Gaia-X Bundle repository will be created and maintained under the EDC project, which can be used as a blueprint for other configurations. Any other future framework would have to develop, maintain and publish its own extensions.

Reply:
+1 on the concept. One thing we should think about is the best place for "code" extensions vs. BOMs. For example, let's assume we have:

1. A policy extension that includes a policy function to evaluate a Gaia-X self-description
2. An Identity Hub extension that handles the storage and rendering of Gaia-X self-description
3. Two Maven BOMs that define launcher configurations. The first configuration is for a connector that includes the policy extension. The second is for an identity hub  that includes the storage/rendering extension

I think those extensions should be placed in different locations:

1. The policy extension should go in the base EDC repo
2. The Identity Hub extension should go into the  IH repo
3. The BOM should be included as Gradle projects in the new "bundle" repository

Do you agree, and can you add this to the ADR?

Reply:
Following up, if you want to leave the policy module in the new (Gaia-X) repo, I'm fine with that or the other way (in core)